packages/wrangler/src/kv/helpers.ts

```diff
@@ -199,7 +199,10 @@ export function unexpectedKVKeyValueProps(keyValue: KeyValue): string[] {
 function asFormData(fields: Record<string, unknown>): FormData {
 	const formData = new FormData();
 
-	for (const [name, value] of Object.entries(fields)) {
+	for (let [name, value] of Object.entries(fields)) {
```

Comment:
Trips up the prefer-const eslint line, not sure if we want to make the function a bit messier with an if/else or disable the rule for this line.

Reply:
Let's go with:

```ts
for (const [name, value] of Object.entries(fields)) {
  formData.append(name, Buffer.isBuffer(value) ? new Blob(value) : value);
}
```